python/metatensor-torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -0,0 +1,341 @@
+import os
+import pathlib
+from typing import Dict, List, Union
+
+import numpy as np
+import torch
+
+from .. import Labels, TensorBlock
+from . import MetatensorAtomisticModule, ModelOutput, ModelRunOptions, System
+
+
+import ase  # isort: skip
+import ase.neighborlist  # isort: skip
+from ase.calculators.calculator import (  # isort: skip
+    Calculator,
+    InputError,
+    PropertyNotImplementedError,
+    all_properties as ALL_ASE_PROPERTIES,
+)
+
+
+# import here to get an error early if the user is missing metatensor-operations
+from .. import sum_over_samples_block  # isort: skip
+
+FilePath = Union[str, bytes, pathlib.PurePath]
+
+
+class MetatensorCalculator(Calculator):
+    """
+    The :py:class:`MetatensorCalculator` class implements ASE's ``Calculator`` API using
+    metatensor atomistic models to compute energy, forces and any other supported
+    property.
+
+    This class can be initialized with any `MetatensorAtomisticModule`, and used to run
+    simulations using ASE's MD facilities.
+    """
+
+    def __init__(
+        self,
+        model: Union[
+            FilePath,
+            torch.jit.RecursiveScriptModule,
+            MetatensorAtomisticModule,
+        ],
+        check_consistency=False,
```

Comment:
I'd say this is a bit different: `check_consistency` is used to check that metatensor-atomistic-models standard is respected (no missing neighbors list, right species, right output shape & metadata) and give a nice error message in case they are not.

The operations checks are there for more "unsafe" checks: skipping them might produce wrong results. Here you'll get a weird error deep down in the model instead. 

We might look for a way to unify both (with some form of check severity maybe?), but since the operations checks are not used for now, I would leave it for a later PR.